Jellyfin.Plugin.Intros/Configuration/config.html

```diff
@@ -14,23 +14,60 @@
+                            <option value="443081835">Cyberpunk Loading</option>
+                            <option value="443081756">Cyberpunk</option>
+                            <option value="443404417">Dark Matter</option>
+                            <option value="443081683">Demolition</option>
+                            <option value="443076427">Digital Glitch</option>
+                            <option value="443404395">Discowave</option>
                             <option value="443081932">Dubstep</option>
+                            <option value="440785270">DVD Coming Soon</option>
+                            <option value="440791181">DVD Feature Presentation</option>
                             <option value="443125346">Elegant Short</option>
-                            <option value="440791181">Retro DVD Intro</option>
-                            <option value="440793415">Alamo Drafthouse No Talking</option>
-                            <option value="440978850">Alamo Drafthouse Shush</option>
-                            <option value="442118203">IT Crowd Anti-Piracy</option>
+                            <option value="443076495">Energy Glitch</option>
+                            <option value="443082249">Fire Explosion</option>
+                            <option value="443381171">Glitch Freeloader</option>
+                            <option value="443381220">Glitch</option>
+                            <option value="443076410">Google Search (Clean)</option>
+                            <option value="443076406">Google Search (Explicit)</option>
+                            <option value="443381155">Liquid Glitch</option>
+                            <option value="443081704">Magical Smoke Reveal</option>
+                            <option value="443380983">Modern</option>
+                            <option value="443404335">Neon Blue</option>
+                            <option value="443076479">Neon Particles</option>
+                            <option value="443404299">Neon Rain</option>
+                            <option value="443381025">Neon Reflection</option>
+                            <option value="443081757">Neon Shock</option>
+                            <option value="443380999">Neon Sign Reflection</option>
+                            <option value="443404077">Neon Storm</option>
+                            <option value="442981865">Netflix</option>
                             <option value="443510978">No More!</option>
-                            <option value="440785270">Retro DVD Coming Soon</option>
-                            <option value="441615803">COVID-19 Social Distancing</option>
+                            <option value="443081763">Paint Splash</option>
+                            <option value="443404007">Plasma</option>
+                            <option value="443076414">Ribbon</option>
+                            <option value="443403981">Sci-fi</option>

```

Comment:
```suggestion
                            <option value="443403981">Sci-Fi</option>

```

Reply:
Super minor change but I prefer capitalizing the latter half of hyphenated words as well.